CONTRIBUTORS.md

```diff
@@ -264,6 +264,13 @@ Listed in alphabetical order.
     </td>
     <td></td>
   </tr>
+  <tr>
+    <td>Arnav Choudhury</td>
+    <td>
+      <a href="https://github.com/arnav13081994">arnav13081994</a>
+    </td>
+    <td></td>
+  </tr>
```

Comment:
This file is now automatically generated, there was no need to update it. 

It's a bit new so this step might still be mentioned somewhere, let me know if this is the case, we should update it.

Reply:
My bad. I thought this was supposed to be done manually. Should I remove that edit then and re-commit?